app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def show_errors(object, field)
+    if object.errors.any?
+      if !object.errors.messages[field].blank?
+        content_tag(:p, class: "error") do
+        field.to_s.capitalize + " " + object.errors.messages[field].first
+        end
+      end
+    end
+  end
 end
```

Comment:
This is different from what I found, but great. Thanks for making this work, it's definitely not an easy problem to tackle. I go over this with you in person tomorrow to make sure I totally understand how it's working.